$MirOS: ports/lang/tcl/8.4/patches/patch-unix_tcl_m4,v 1.1 2006/11/20 21:08:11 tg Exp $
_MirOS: ports/lang/tcl/8.4/patches/patch-unix_tcl_m4,v 1.1 2006/11/20 21:08:11 tg Exp $
--- unix/tcl.m4.orig	Thu Oct 19 17:27:11 2006
+++ unix/tcl.m4	Mon Nov 20 21:39:24 2006
@@ -1471,7 +1471,7 @@ dnl AC_CHECK_TOOL(AR, ar)
 	    UNSHARED_LIB_SUFFIX='${TCL_TRIM_DOTS}\$\{DBGX\}.a'
 	    TCL_LIB_VERSIONS_OK=nodots
 	    ;;
-	OpenBSD-*)
+	OpenBSD-*|MirBSD-*)
 	    case `arch -s` in
 	    m88k|vax)
 		SHLIB_CFLAGS=""
@@ -2033,7 +2033,7 @@ dnl AC_CHECK_TOOL(AR, ar)
 		    ;;
 		IRIX*)
 		    ;;
-		NetBSD-*|FreeBSD-*|OpenBSD-*)
+		NetBSD-*|FreeBSD-*|OpenBSD-*|MirBSD-*)
 		    ;;
 		Darwin-*)
 		    ;;
